snapshotter/create_snapshot.sh

```diff
@@ -51,7 +51,7 @@ done
 
 # Only upload snapshots if they are all valid.
 if [ "$ARE_SNAPSHOTS_VALID" -eq "1" ]; then
-    gsutil cp "$TMPDIR/snap-$BLOCK_HEIGHT" "gs://$SNAPSHOT_BUCKET/snap-$BLOCK_HEIGHT"
+    gsutil cp "$TMPDIR/snap-$BLOCK_HEIGHT" "gs://$SNAPSHOT_BUCKET/snap-$BLOCK_HEIGHT.gz"
```

Comment:
from what i have read from the code, gz should be part of the file already. [code here](https://github.com/helium/blockchain-core/blob/2022.05.12.0/src/blockchain.erl#L2045)